openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -518,6 +527,11 @@ def _tile_render(self, payload):
 
         instance.data["assemblySubmissionJobs"] = assembly_job_ids
 
+        # Remove config files to avoid confusion about where data is coming
+        # from in Deadline.
+        for config_file in config_files:
+            os.remove(config_file)
```

Comment:
We're also passing a config to `aux_files` for the payload. How do these config files differ from those? Should we have better variable naming distinction?

Reply:
Those are the same config files, but the Draft Tile Assembler plugin only gets the config file from `aux_files`, so having it in the plugin parameters or on disk, actually makes more confusion.